data/ehentai

```diff
@@ -3,4 +3,6 @@ ehgt.org
 ehtracker.org
 ehwiki.org
 exhentai.org
+hath.network
+hentaiathome.net
```

Comment:
Would you please share more info about the `hentaiathome.net` domain?
I need to verify it belongs to the "official", and know where this domain is serving.

Reply:
`hentaiathome.net` is listed in [EHWiki](https://ehwiki.org/wiki/IPs), and used in the [Hentai@Home](https://ehwiki.org/wiki/Hentai@Home) system. The latest source code of Hentai@Home is available at <https://repo.e-hentai.org/hath/HentaiAtHome_1.6.1_src.zip>.